pytorch_cnn_trainer/plotter.py

```diff
@@ -0,0 +1,12 @@
+import matplotlib.pyplot as plt 
+
+def plot(history,metric1,metric2):
```

Comment:
We should probably give a better name 🙂

Reply:
I probably don't know any better than this

Reply:
Can your rename it from plot to `plot_results`
It will clash with `plt.plot`